### DISCUSSION TITLE
latest release is from june 2023? (tarmworth)

### DISCUSSION THREAD

**Discussion by markat1** [UglyToad/PdfPig#872] [Q&A] [answered]
This is really a great project, but I hope it's okay asking the following.

I was wondering why latest release is 1 year old [(tarmworth)-](https://github.com/UglyToad/PdfPig/releases/tag/v0.1.8
) how come? or am I missing something?

**Comment by EliotJones** [comment, marked as answer]
Please see discussion at https://github.com/UglyToad/PdfPig/issues/842

**Reply by markat1** [reply]
Sorry @EliotJones  - didn't see that discussion, when I searched in the discussion section! Thanks for the reply!
